common/persistence/workflowStateCloseStatusValidator_test.go

```diff
@@ -21,9 +21,10 @@
 package persistence
 
 import (
-	"github.com/uber/cadence/common/types"
 	"testing"
 
+	"github.com/uber/cadence/common/types"
```

Comment:
nit: all cadence import will be moved to the very end and have a separate section if you remove the empty line in L27 and run make fmt

Reply:
Thanks